core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -316,6 +332,25 @@ public ServerBuilder port(ServerPort port) {
         return this;
     }
 
+    /**
+     * Adds an {@link ChannelOption} to use in netty {@link ServerBootstrap} to create new {@link Server}.
+     * Note that when you put same option multiple times it will be overwritten.
```

Comment:
Note that the previously added option will be overridden if the same option is set again.